### ISSUE TITLE
Clearing Flags Other Than "Skybox" Or "Solid Color"

### ISSUE THREAD

**Issue by TheMasonX** [Unity-Technologies/PostProcessing#3]
First of, all, want to say that you guys have done an incredible job, and I am super excited for more/better image effects.

The issue comes whenever using a dual camera setup where the second one draws on top of the first by only clearing the depth (in my case, using a second camera to render distant objects that would otherwise create precision issues if using a single camera with a large far clip). This works fine with all the old image effects, but with the new stack implementation, most of them lead to erratic behavior.

Usually it's the standard trail issue as though rendering a single camera but not clearing depth/anything, though it's done two other things as well on rare occasions, mainly what looks like feedback loops where the background either quickly trends towards solid white, or I've only seen once and didn't get a screenshot, the colors bloom out and dominate local regions, resembling a limited color palette image. These other results pop up seemingly randomly, as no changes to the scene or anything are necessary.

Gamma vs linear has no effect, though in forward at least, only AO and Motion Blur don't work, both of which enter a feedback loop, and turn dark grey and white respectively after about a half second. They both work with clearing flags changed to skybox or solid color though. In deferred, it doesn't matter which one(s) are active, as any of them will trigger the issue, whereas having none selected, but the MonoBehaviour still on and a profile ScriptableObject assigned works fine.

Usual Trail effect:
https://gyazo.com/f7ef133698ff2c8e9849b5776a73ead0

One Rare Feedback Loop:
https://gyazo.com/9520feaa6ddcd25e6b654c7a71a968a4

Normal For Comparison (still very early in development):
https://gyazo.com/16c99377a45fce19c387da9e16647e66

If you need anything more, I'm happy to provide anything necessary :) Thanks for your time and effort, I really appreciate it!

**Comment by Chman** [comment]
Is MSAA enabled ? [We have a bug currently](https://github.com/Unity-Technologies/PostProcessing/wiki#known-issues) with AO and Motion Blur when MSAA is enabled with the Forward rendering path. This is fixed in Unity 5.6 (@stramit did a big refactor on this part). HDR + MSAA + Stacked cameras will work properly then. It's too risky a change to bring to 5.5 at this stage unfortunately.

As for the multi camera issue, I'll do some testing but I'm pretty sure it's fixed on 5.6 too.

**Comment by TheMasonX** [comment]
@Chman No, I had it turned off because TAA said it isn't compatible. As an update, it seems to be caused by making any changes to profile, but works after restarting the editor. It had been working consistently after switching to forward and back, but that has stopped working even after multiple restarts.

This is profile specific, as switching over to a profile that's either brand new or hasn't been updated since starting Unity works, though switching back causes it to be broken again, so the issue is probably with some data in the scriptable object that gets modified when edited, and then regenerated on loading.

Also, whatever this edited data is, it's now causing forward to go white on play (same issue, as the same conditions that make deferred streak now cause white in forward). This is really odd, as it was working the other day. Ahh, the joys of software development...

**Comment by simeonradivoev** [comment]
I have exactly the same problem. For me it happens only after I change a setting. Any setting.

**Comment by tbg10101** [comment]
+1 for this issue, but I can hold out until 5.6 if needed.

**Comment by Chman** [comment]
Reopening this - wrong issue #id in commit message closed it automatically.

**Comment by Kink3d** [comment]
Fixed in my tests in 5.6. Reopen if reproducible there (Cannot backport to 5.5 unfortunately).
